src/main/java/com/sergiofah/integracao/model/Line.java

```diff
@@ -0,0 +1,42 @@
+
+package com.sergiofah.integracao.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "id",
+        "line"
+})
+public class Line {
+
+    @JsonProperty("id")
```

Comment:
`@JsonProperty` is not necessary here. Please, remove it and do the same thing for the other unnecessary properties.